posthog/migrations/0318_alter_earlyaccessfeature_stage.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.18 on 2023-06-02 14:28
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0317_batch_export_models"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
while you're doing this migration, might want to add the archived option support as well?